connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -345,6 +565,14 @@ public void fenceZombies(final @PathParam("connector") String connector,
     @GET
     @Path("/{connector}/tasks/{task}/status")
     @Operation(summary = "Get the state of the specified task for the specified connector")
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
+                    content = @Content(mediaType = "application/json", array = @ArraySchema(schema = @Schema(implementation = ConnectorStateInfo.TaskState.class))),
+                    description = "Task status fetched successfully for specified connector"
```

Comment:
We've used `retrieved` everywhere so let's stick to that instead of introducing `fetched` here. We can also drop the `for specified connector` ending.